### PR TITLE
[WIP][YUNIKORN-2540] clean up constants in test case

### PR DESCRIPTION
**Pull request by amikai**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - http://yunikorn.apache.org/community/how_to_contribute   

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [x] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/YUNIKORN/
* Put link here, and add [YUNIKORN-*Jira number*] in PR title, eg. `[YUNIKORN-2] Gang scheduling interface parameters`

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Comment by pbacsko** [comment]
@amikai it's an useful change, I like it.

Two things to keep in mind:
1) We're still working on 1.5.1, so larger refactor changes should be hold off until the contents of the release are finalized.
2) Please fill the PR template properly, at least the "What is this PR for?" and the "What is the Jira issue?" part. Thanks.

**Comment by chia7712** [comment]
@amikai Could you please fix conflicts?

File: pkg/cache/context_test.go [hunk 1]
```diff
@@ -49,16 +49,46 @@ import (
 )
 
 const (
-	Host1  = "HOST1"
-	appID1 = "app00001"
-	appID2 = "app00002"
-	appID3 = "app00003"
-)
+	// NOTE: why this variable is public of package
```
Comment (ryankert01) [inline pkg/cache/context_test.go:52]: I don't think it should be public, but the name `host1` will be duplicate to this variable:
https://github.com/apache/yunikorn-k8shim/blob/741c0d801ac4530669b8850706efe3f0bc0d5718/pkg/cache/context_test.go#L610-L611

So, a rename is needed.

**Review by ryankert01 — COMMENTED** [review]
In-line comment regarding `L52: // NOTE: why this variable is public of package`.

**Comment by pbacsko** [comment]
@ryankert01 please fill out the PR template (at least the "What is this PR for?" and "What is the Jira issue?" sections), resolve conflicts by merging master and drop the Draft status.

**Comment by ryankert01** [comment]
@pbacsko I don't think it's my PR.

**Comment by pbacsko** [comment]
Oh sorry. I wanted to ping @amikai.

**Comment by amikai** [comment]
Due to the lack of time recently, close the PR and make the issue unassigned.

**Pull request closed without merging**